app-config.yaml

```diff
@@ -12,7 +12,7 @@ backend:
     connection: ':memory:'
   cors:
     origin: http://localhost:3000
-    methods: [GET, POST, PUT, DELETE]
+    methods: [GET, POST, PUT, DELETE, PATCH]
```

Comment:
Would suggest we may not want to change this globally for the example app, but if `PATCH` method is needed adding it to the README for the plugin setup instructions?